services/pom.xml

```diff
@@ -27,6 +27,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
```

Comment:
This is not necessary and can be removed (it was only needed in older wildfly/eap versions).